Sources/Apollo/GraphQLOperation.swift

```diff
@@ -1,10 +1,19 @@
+/// Enumeration of the possible GraphQL operations that can be executed against an API endpoint.
 public enum GraphQLOperationType {
   case query
   case mutation
   case subscription
+
+  var cacheKey: String {
```

Comment:
i would call this `rootCacheKey` to more clearly emphasize that this is not the whole cache key.